index.bs

```diff
@@ -160,7 +169,15 @@ The {{PerformanceElementTiming/naturalHeight}} attribute must return the value i
 
 The {{PerformanceElementTiming/id}} attribute's getter must return <a>element</a>'s {{Element/id}} attribute value.
 
-The {{PerformanceElementTiming/element}} attribute's getter must return <a>element</a> if <a>element</a> is <a>connected</a>, and must return <code>null</code> otherwise.
+The {{PerformanceElementTiming/element}} attribute's getter must run these steps:
+<div algorithm="PerformanceElementTiming element">
+    1. Let |element| be the <a>context object</a>'s <a>element</a>.
+    1. If |element| is not <a>connected</a>, return <code>null</code>.
+    1. Let |window| be the <a>context object</a>'s <a>relevant global object</a>.
```

Comment:
Nit: I think you can immediately get to the responsible document of the relevant env settings object, rather than go through window

Reply:
Yes that sees to be an option too, but how would I get to the environment settings object directly?

Reply:
The relevant settings object gives you that, I think: https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object

Reply:
Done. Seems like the same number of steps though!